src/main/resources/db/migration/V4__document_update.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE document ADD COLUMN organization_uuid UUID;
```

Comment:
Don't rename migration files because it will break current migrations and database on demo and staging.